web/controllers/survey_controller.ex

```diff
@@ -288,14 +288,10 @@ defmodule Ask.SurveyController do
         send_resp(conn, :bad_request, "")
 
       _ ->
-        multi = Multi.new
-        |> Multi.delete(:survey, survey)
-        |> Multi.insert(:log, ActivityLog.delete_survey(project, conn, survey))
-        |> Repo.transaction
-
-        case multi do
+        case SurveyAction.delete(survey) do
           {:ok, _} ->
             project |> Project.touch!
+            ActivityLog.delete_survey(project, conn, survey) |> Repo.insert!
```

Comment:
Why move out the activity log insertion outside the transaction?
I would expect this to happen inside the SurveyAction.delete